### PR TITLE
[OT210-46] Endpoint Creación Novedades

### PR DESCRIPTION
**Pull request by mike47k**
## Why is this change required?

COMO usuario administrador
QUIERO agregar una novedad en el sistema
PARA poder informar las Novedades de la ONG

Criterios de aceptación:
POST /news - Deberá validar la existencia de los campos enviados, para almacenar el registro en la tabla News. Antes de almacenarla, deberá asignarle la columna category con el valor "news".

### Type of change
- [x] New feature
- [ ] New Tests
- [ ] Bug fix
- [ ] Refactor

### Checklist

- [ ] Traceability between this change and Jira.
- [ ] ```mvn clean install``` was run and all tests completed successfully.
- [ ] There are no unused variables and/or imports in the modified classes.
- [ ] There are no imports in the modified classes with the wildcard character. Ex: ```com.somepackage.*```.
- [ ] Slf4j was used for the application log instead ```System.out```.
- [ ] Public methods are documented with ```javadoc```.

### CODE REVIEW

**Comment by mike47k** [comment]
postmanCreateNews
dbNewCreated
Captura desde 2022-06-10 10-42-01

File: src/main/java/com/alkemy/ong/domain/usecase/impl/NewServiceImpl.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+@Service
+@RequiredArgsConstructor
+public class NewServiceImpl implements NewService {
+
+    private final NewRepository newJpaRepository;
+
+    private final CategoryRepository categoryJpaRepository;
+
+    @Value("${default.role.id}")
```
Comment (manerajona) [inline src/main/java/com/alkemy/ong/domain/usecase/impl/NewServiceImpl.java:21]: agregar una nueva variable en `application.properties`. Ex: `default.category.id`

File: src/main/java/com/alkemy/ong/domain/usecase/impl/NewServiceImpl.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    private Long defaultCategoryId;
+
+    @Override
```
Comment (manerajona) [inline src/main/java/com/alkemy/ong/domain/usecase/impl/NewServiceImpl.java:24]: Agregar `@Transactional(readOnly = true)` por favor

**Review by manerajona — CHANGES_REQUESTED** [review]

**Comment by mike47k** [comment]
listo

**Pull request merged**